konfera/templates/konfera/conferences.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'konfera/base.html' %}
+{% load i18n %}
+
+
+{% block title %}
+{% trans "Conferences List" %}
+{% endblock %}
+
+
+{% block content %}
+<div class="list-group">
+    {% for conference in object_list %}
+    <a href="{% url 'event_details' conference.slug %}" class="list-group-item">
+        <h4 class="list-group-item-heading"> {{ conference.title }} </h4>
+        <p class="list-group-item-text">{{ conference.date_from|date:"d. F Y – H:m"}} - {{ conference.location}}</p>
```

Comment:
Display date and time from - to (in case event ends the same day), in case event end on different date just show from to dates.
please add empty space before }}